index.js

```diff
@@ -1,65 +1,99 @@
 // ***** Question 1 *****
 // *** Uncomment the lines below to test
-// console.log("%cQuestion 1", "color: red") 
+ console.log("%cQuestion 1", "color: red") 
+console.log("blob")
 
-// console.log(counter) 
+var counter = 1
+var counter = 2
+ 
 // // => 2
-// console.log("%c----------", "color: red") 
+ console.log("%c----------", "color: red") 
 
 
 // ***** Question 2 *****
 // *** Uncomment the lines below to test
 // *** after testing, comment the line causing the error back in 
 // *** otherwise, the error will stop the rest of your code from running
-// console.log("%cQuestion 2", "color: red") 
+ 
+const name = "Shafee"
 
-// console.log(name)
-// => "Raffy"
-// name = "Not Raffy" 
-// => TypeError
-// console.log("%c----------", "color: red") 
+console.log("%cQuestion 2", "color: red") 
+
+console.log(name)
+
+ // => "Raffy"
+ // name = "Not Raffy" 
+ // => TypeError
+ console.log("%c----------", "color: red") 
 
 
 
 // ***** Question 3 *****
 // *** Uncomment the lines below to test
-// console.log("%cQuestion 3", "color: red") 
+ 
+function drinkWater(currentThirstLevel) {
+
+  console.log("Man I sure am thirsty")
+  currentThirstLevel -= 1
+  console.log("Ahh that hits the spot")
+  return currentThirstLevel
+}
```

Comment:
Good! Way to use  [camel case](https://upload.wikimedia.org/wikipedia/commons/thumb/c/c8/CamelCase_new.svg/1200px-CamelCase_new.svg.png) for the function and variable names.